core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java

```diff
@@ -1349,7 +1643,7 @@ public List<SqlNode> createAsFullOperands(RelDataType rowType, SqlNode leftOpera
     selectList.add(node);
   }
 
-  protected void parseCorrelTable(RelNode relNode, Result x) {
+  private void parseCorrelTable(RelNode relNode, Result x) {
```

Comment:
Please make it as protected as this method is used in child classes.

Reply:
I will address suggested changes in next PR